### ISSUE TITLE
Inferno-Program Implantable Device tests are not executing

### ISSUE THREAD

**Issue by ashu-bdvt** [onc-healthit/inferno-program#439]
Thank you for reporting a possible bug in Inferno! Please fill in as much of the template below as you can.

**Subject of the issue**
Inferno Program edition Implantable Device tests are not getting executed. It is stuck at the very first test. Even after waiting for ~ 1hr no progress or no errors

The same test gets executed in my local inferno-program instance(running on docker)

**Your environment**
* Edition of inferno (Community or Program): Program
* Version of inferno: 1.9.0
* Which browser and version(s) is the bug present on?: Chrome

**Steps to reproduce**
Just run the "Single Patient" tests for the below endpoint and patient
Endpoint: https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d
Patient: 12753537
Device: No need to give anything since it works in my local setup without it, however if needed please use 31563397

Include:

* The selected test suite
* The URL of the FHIR server being tested
* The Test ID, if it's a problem with a specific test.
* Any required configuration options (client ID, client secret, etc.)
* Anything else needed to make the issue occur.
Screenshot:
<img width="1536" alt="Screenshot 2022-02-24 at 6 57 27 PM" src="https://user-images.githubusercontent.com/6142355/155538462-546075ce-9c3f-40cb-ae16-c9fad66f0e12.png">

**Expected behavior**
Implantable device tests should run on the remote interno-program instance

**Actual behavior**
Implantable device tests is not running on the remote interno-program instance

**Comment by ashu-bdvt** [comment]
Attaching local validation where these test are passing
<img width="1108" alt="Screenshot 2022-02-24 at 7 53 55 PM" src="https://user-images.githubusercontent.com/6142355/155542400-9bd4265e-6819-4aed-85e2-daecd7b4981d.png">

**Comment by yunwwang** [comment]
Hello @ashu-bdvt 
Can you provide test-id for your test session? Thanks

**Comment by ashu-bdvt** [comment]
How do i get tht? I just cancelled the sequence and saved the instance at https://inferno.healthit.gov/inferno/3uo33ovEqrn/

**Comment by yunwwang** [comment]
Can you try testing it with a valid token?

**Comment by ashu-bdvt** [comment]
That's an interesting find. I also see the same special character via postman. However when i run against the same endpoint in my local inferno instance, it's just a plain empty string and the tests are passing. Do you think it's what causing the failure?
<img width="1157" alt="Screenshot 2022-02-25 at 2 12 07 PM" src="https://user-images.githubusercontent.com/6142355/155683718-d94d51e3-09f6-4559-83a0-6272bd3e7052.png">

**Comment by yunwwang** [comment]
What I observed is that the state shows Device's ID which means Inferno did get that Device resource but something happened and crashed the process. 
![image](https://user-images.githubusercontent.com/8761175/155732973-4ca731b3-f084-427c-89f7-50c283de3d9c.png)

My first though was that the Device resource was invalid.

The resource you posted is strange. If the Device.type.text is empty, why do you have that element there?

**Comment by yunwwang** [comment]
@ashu-bdvt Our investigation indicate that you many send this in your Device resource
```
"type": {
  "text": "\u0000"
}
```
Though it is valid in FHIR, it is highly not recommended because it does not carry any meaning. If you don't have type, you should follow the US Core [Missing Data](http://hl7.org/fhir/us/core/STU3.1.1/general-guidance.html#missing-data) section

**Comment by arscan** [comment]
I do not think we will fix this in v1.x, because there doesn't seem to be a good reason to send characters like this and we are not planning on any releases on 1.x.  However, we could fix it in [v2.x](https://github.com/onc-healthit/onc-certification-g10-test-kit) if you are seeing this over there as well. Have you had a chance to move over to that new version?
